docs/api/commands/visit.mdx

```diff
@@ -354,9 +354,24 @@ cy.visit('index.html').then((contentWindow) => {
 
 ### User agent
 
-Trying to change the `User-Agent`? You can set the `userAgent` as a
+Trying to change the `User-Agent`?
+
+You can set the `userAgent` as a
 [configuration value](/guides/references/configuration#Browser) in your
-configuration file.
+configuration file or for a **single** `cy.visit()` by setting the User-Agent
+header in the options.
+
+```javascript
+// You can check your user-agent using a third-party tool
+cy.visit('https://www.whatsmyua.info/', {
+  headers: {
+    // override user-agent string
+    'user-agent': 'MyBrowser/1.0 (CoolOS; v2)',
+  },
+})
+```
+
```

Comment:
This is missing the caveats to this approach.

Reply:
removed example